models/ed/R/veg2model.ED2.R

```diff
@@ -1,10 +1,18 @@
 #' Writes ED specific IC files
-#'
+#' @title veg2model.ED2
+#' @name veg2model.ED2
```

Comment:
But while I'm here, *thank you* for all your work adding missing `@param` entries in this and previous PRs. It has cleaned up a *lot* of package check warnings and made the documentation much easier to understand. I'll take an occasional extra title tag over missing parameter descriptions any day!

Reply:
Thanks for the helpful feedback on the use of @title and @name. I am still learning how this all fits together. In my limited experience I have found that having the @name is helpful for NAMESPACE calls for the function. When I was going through ic_process line-by-line I was running into some errors where R would tell me it could not find function even with a namespace call. I found that when this would happen it was usually because the function did not have the @name param or because the @name param did not match the function name.

Reply:
> I was running into some errors where R would tell me it could not find function even with a namespace call

High-level point first: If the missing function is present in your package's NAMESPACE file you can be confident that Roxygen has done its job correctly and the issue lies elsewhere. In particular, my first stop for debugging function-not-found issues is to check the namespace *in the installed copy of the package* using `ls(getNamespace("pkgname_here"))`, and if the function is missing there I reinstall the package and try again. Even after a decade of building R packages, the overwhelming majority of my "why can't R find the function I know I added" problems turn out to be "I did not in fact reinstall the package after adding the function" problems.

Lower-level, I don't see how `@name` would affect function availability, because Roxygen only uses it change the name given in the documentation pages and not the name it exports in the namespace. Consider the following code file:

```
#' Rd name implicitly taken from function name
#' @export
foo1 <- function(){}

#' Rd name given explicitly but matches function name
#' @name foo2
#' @export
foo2 <- function(){}

#' Rd name does not match function name
#' @name bar
#' @export
foo3 <- function(){}
```

Once processed with Roxygen, this generates files documentation files `man/foo1.Rd`, `man/foo2.Rd`, and `man/bar.Rd` (which last will contain `\name{bar}` but also `\alias{foo3}` so that people typing `?foo3` can find the right help page), but the NAMESPACE file will be:

```
export(foo1)
export(foo2)
export(foo3)
```

Note that there's no `export(bar)` and that `pkgname::bar()` will still return an error.